schemas/dbt/manifest/v9.json

```diff
@@ -44,6 +44,9 @@
           {
             "$ref": "#/definitions/SnapshotNode"
           },
+          {
+            "$ref": "#/definitions/UnitTestNode"
```

Comment:
let's cut a manifest v11 for these changes?

Reply:
I'm updating to v11 manifest in another branch, because there's no point in doing that in multiple places. Will update to create a modified v11 once that's merged.

Reply:
I removed the updates to the v9 manifest json schema. I decided to leave the v11 manifest without changes from this branch, because it doesn't look like they're actually needed at this point and we are probably going to not include UnitTestNodes in the current manifest definition, so might not be needed in the end anyway.